content/docs/command-reference/config.md

```diff
@@ -205,7 +205,7 @@ to learn more about the state file (database) that is used for optimization.
 > [Create a Bucket](https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html).
 
 ```dvc
-$ dvc remote add myremote s3://bucket/path
+$ dvc remote add myremote s3://bucket/key
```

Comment:
why do we change this? it's confusing for most people.

Reply:
Ok, rolled these back. Maybe it should be `path/to/data` or `path/to/object` depending on the type I think.

Reply:
p.s. There were instances of `bucket/mykey` before too, so given this comment I changed those to `path` also (88d5f270).